client/state/selectors/is-automated-transfer-active.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import {
+	flowRight as compose,
+	includes,
+} from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { transferStates } from 'state/automated-transfer/constants';
+import { getAutomatedTransferStatus } from 'state/automated-transfer/selectors';
+
+/**
+ * Maps automated transfer status value to indication if transfer is active
```

Comment:
nope. maps _noun_ (status) to _noun_ (indication)

we can change it if you think it's odd